### ISSUE TITLE
After updating Anaconda 3 via Anaconda Navigator the Navigator disappears

### ISSUE THREAD

**Issue by ArthurZ** [ContinuumIO/anaconda-issues#1280]
After updating Anaconda 3 via Anaconda Navigator the Navigator disappears among many other tools.
The shortcuts used to reside in the Windows Start -
`C:\Users\MyName\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Anaconda3 (64-bit)`
For reference I am on Windows 10.

**Comment by goanpeca** [comment]
This is something that has been already reported @csoja.

I am not sure what changed @ilanschnell.

**Comment by goanpeca** [comment]
There is a new release available for Anaconda. You can download it from https://www.continuum.io/downloads

This release includes Anaconda Navigator 1.4.3 which includes several fixes and improvements.

You can update it by typing in the command prompt
`$ conda install anaconda-navigator`

**Comment by ArthurZ** [comment]
Thank you for sharing the news @goanpeca!
I first tried to upgrade the Anaconda 3 installation on Windows, but the installer did not allow to proceed beyond the point 
```---------------------------
Anaconda3 4.3.0 (64-bit) Setup
---------------------------
Directory 'C:\Users\arthu\Anaconda3' already contains a Python installation,
please choose a different location.
---------------------------
OK   
---------------------------
```
I did not see upgrade instructions listed. And I did not want to end up un-installing or creating an extra version of Anaconda. Is there an in-place upgrade option?
As to the anaconda-navigator itself, it now works 👍 
Interestingly, it allows to install RStudio 1.0.44 as one of the options even though I already have it installed (ver. 1.0.136 which I checked is the latest).
